From: Yu-Chun Lin <eleanor15x@gmail.com>
Subject: Documentation/core-api: min_heap: update for variable types change
Date: Sat, 15 Feb 2025 23:54:21 +0800

Update the documentation to reflect the change in variable types of 'nr'
and 'size' from 'int' to 'size_t', ensuring consistency with commit
dec6c0aac4fc ("lib min_heap: Switch to size_t").

Link: https://lkml.kernel.org/r/20250215155421.2010336-1-eleanor15x@gmail.com
Signed-off-by: Yu-Chun Lin <eleanor15x@gmail.com>
Acked-by: Kuan-Wei Chiu <visitorckw@gmail.com>
Cc: Ching-Chun (Jim) Huang <jserv@ccns.ncku.edu.tw>
Cc: Jonathan Corbet <corbet@lwn.net>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 Documentation/core-api/min_heap.rst |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/Documentation/core-api/min_heap.rst~documentation-core-api-min_heap-update-for-variable-types-change
+++ a/Documentation/core-api/min_heap.rst
@@ -47,8 +47,8 @@ Example:
 
     #define MIN_HEAP_PREALLOCATED(_type, _name, _nr)
     struct _name {
-        int nr;         /* Number of elements in the heap */
-        int size;       /* Maximum number of elements that can be held */
+        size_t nr;         /* Number of elements in the heap */
+        size_t size;       /* Maximum number of elements that can be held */
         _type *data;    /* Pointer to the heap data */
         _type preallocated[_nr];  /* Static preallocated array */
     }
_
